shotover-proxy/examples/redis-cluster-ports-rewrite/topology.yaml

```diff
@@ -6,7 +6,7 @@ sources:
 chain_config:
   redis_chain:
     - RedisClusterPortsRewrite:
-         new_port: 2004
+         new_port: 6379
```

Comment:
Wouldn't this do nothing? I thought the ports were already 6379? 🤔

Reply:
Yeah I wanted the example in the docs to work though. I should change it back for the tests though

Reply:
It still did do something on 6379 right? because all the ports expose by docker are at 2220-2225.
So maybe we should actually use 6379 to make the example more realistic?

Reply:
Thinking about this again.
If redis is behind a docker that maps 6379:2001 then doesnt redis think it is behind port 6379?
To actually test port remapping we would have to set the redis config to run on some unusual port.

Reply:
What do you mean by "redis think it is behind port 6370"?

Reply:
For the redis-cluster-ports-rewrite integration test the redis port gets remapped as follows:
redis: 6379
docker: 2220
shotover: 6379

Because redis and shotover are both configured with the same port number the integration test doesnt test the remapping as its remapping 6379 -> 6379.

You can verify that redis thinks its on 6379 by directly connecting to a redis instance in the redis-cluster-ports-rewrite cluster.
![image](https://user-images.githubusercontent.com/5120858/146300593-e18fe51d-e358-4cec-9359-c73087e70060.png)

Reply:
Shotover is running on 6380

Reply:
The docs are now wrong though, I'll correct them.

Reply:
Ahhh so it is!
Not sure how I got confused, maybe it was the docs as you mentioned...